src/main/java/com/minecolonies/coremod/client/model/ModelEntityBakerMale.java

```diff
@@ -86,28 +86,7 @@ public ModelEntityBakerMale()
         bipedHeadwear.isHidden = true;
     }
 
-    @Override
-    public void render( final Entity entityIn, final float limbSwing,
-            final float limbSwingAmount,
-            final float ageInTicks,
-            final float netHeadYaw,
-            final float headPitch,
-            final float scaleFactor
-    )
-    {
-        setRotationAngles(limbSwing, limbSwingAmount, ageInTicks, netHeadYaw, headPitch, scaleFactor, entityIn);
-        bipedHead.render(scaleFactor);
-        bipedBody.render(scaleFactor);
-        bipedLeftArm.render(scaleFactor);
-        bipedRightArm.render(scaleFactor);
-        bipedRightLeg.render(scaleFactor);
-        bipedLeftLeg.render(scaleFactor);
-        base.render(scaleFactor);
-        middle.render(scaleFactor);
-        top.render(scaleFactor);
-    }
-
-    private void setRotation(final ModelRenderer model, float x, float y, float z)
+    private void setRotation(final ModelRenderer model, final float x, final float y, final float z)
```

Comment:
This whole method can be removed now, child pieces will be given the same rotation as their parents.